### ISSUE TITLE
TypeError: e.with is not a function

### ISSUE THREAD

**Issue by Logerfo** [microsoft/vscode#84688]
## Issue
The API is crashing with the following line of code:
```ts
vscode.workspace.getConfiguration("open-all-files", args._fsPath)
```
[Source](https://github.com/Logerfo/open-all-files/blob/master/src/extension.ts#L35).

> 
```
TypeError: e.with is not a function
at T.getFolder (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:350:909)
	at Object.getConsolidatedConfigurationModelForResource (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:339:816)
	at Object.getConsolidateConfigurationModel (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:339:572)
	at Object.getValue (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:336:116)
	at w.getConfiguration (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:668:743)
	at Object.getConfiguration (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:712:550)
	at openAllFiles (c:\Git\open-all-files\src\extension.ts:35:44)
	at v._executeContributedCommand (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:527:863)
	at v.$executeContributedCommand (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:528:217)
	at f._doInvokeHandler (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:646:285)
	at f._invokeHandler (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:645:999)
	at f._receiveRequest (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:644:664)
	at f._receiveOneMessage (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:643:477)
	at c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:641:754
	at l.fire (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:47:207)
	at _.fire (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:187:274)
	at c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:798:104
	at l.fire (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:47:207)
	at _.fire (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:187:274)
	at Object._receiveMessage (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:191:629)
	at c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:188:824
	at l.fire (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:47:207)
	at f.acceptChunk (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:184:737)
	at c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:184:89
	at Socket.t (c:\Users\Bruno\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:193:68)
	at Socket.emit (events.js:200:13)
	at addChunk (_stream_readable.js:294:12)
	at readableAddChunk (_stream_readable.js:275:11)
	at Socket.push (_stream_readable.js:210:10)
	at Pipe.onStreamRead (internal/stream_base_commons.js:166:17)
```
This might have something to do with #80719.
## Info
Version: 1.40.0 (user setup)
Commit: 86405ea23e3937316009fc27c9361deee66ffbf5
Date: 2019-11-06T17:02:13.381Z
Electron: 6.1.2
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18362

**Comment by jrieken** [comment]
the second argument must be a URI not a string

**Comment by Logerfo** [comment]
@jrieken Thank you for your answer. I think you meant the lines 8403-8415: https://github.com/microsoft/vscode/blob/c408a2c02d78982b1f83819c587c02bb59af9f93/src/vs/vscode.d.ts#L8403-L8415
Both webpack and tsc were showing no problems with the current code whatsoever. Either way, fixed with Logerfo/open-all-files@636e5e07ab338727ee4e287890c87c6a9247ced8
